validation/parser/face_vector_fill_parser.py

```diff
@@ -0,0 +1,19 @@
+from functools import partial
+from parser.container_parser import ContainerParser
+from parser.face_vector_parser import FaceVectorParser
+from parser.pipeline.parser_pipeline_funcs import fill_empty
+
+
+class FaceVectorFillParser(FaceVectorParser):
+    def __init__(self,
+                 container_parser: ContainerParser,
+                 embedding_size: int,
+                 distance_metric: str) -> None:
+        super().__init__(container_parser, distance_metric)
+        self._embedding_size = embedding_size
+        self.__parser_pipeline = None
+
+    def _build_pipeline(self) -> None:
+        super()._build_pipeline()
+        partial_fill = partial(fill_empty, embedding_size=self._embedding_size)
```

Comment:
Usage of `functools.partial` can often be rewritten in a more straight forward way using a normal function:

```py
def partial_fill(self, pairs: Iterable[Pair]) -> Iterator[Pair]:
    return fill_empty(pairs, self._embedding_size)

self._parser_pipeline.add(self._partial_fill)
```